optimum/exporters/onnx/model_configs.py

```diff
@@ -632,7 +632,32 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
-class VaeOnnxConfig(ViTOnnxConfig):
+class VaeEncoderOnnxConfig(ViTOnnxConfig):
```

Comment:
Why inherint from `ViTOnnxConfig` if we do not use much from it?

Reply:
Good catch, it should be `VisionOnnxConfig`